src/main/webapp/dev/js/common/bootboxWrapper.js

```diff
@@ -16,15 +16,15 @@ const DEFAULT_YES_TEXT = 'Yes';
 const DEFAULT_NO_TEXT = 'No';
 
 function applyStyleToModal(modal, statusType) {
-    modal.find('.modal-header').addClass(`alert-${statusType || StatusType.DEFAULT}`)
-            .find('.modal-title').addClass(`icon-${statusType || StatusType.DEFAULT}`);
+    modal.find('.modal-header').addClass(`alert-${statusType || BootstrapContextualColors.DEFAULT}`)
+            .find('.modal-title').addClass(`icon-${statusType || BootstrapContextualColors.DEFAULT}`);
 }
 
 /**
  * Custom alert dialog to replace default alert() function
  * Required params: titleText and messageText
  * Optional params: okButtonText (defaults to "OK")
- *                  statusType (defaults to StatusType.DEFAULT)
+ *                  statusType (defaults to BootstrapContextualColors.DEFAULT)
  */
 function showModalAlert(titleText, messageText, okButtonText, statusType) {
```

Comment:
The parameter is no longer `statusType`?

Reply:
@LiHaoTan very true. Apologies for this blunder. Will do it right away.